media/css/firefox/new/berlin.scss

```diff
@@ -0,0 +1,283 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/sections';
+
+@font-face {
+    font-display: swap;
+    font-family: 'Fira Sans Heavy';
+    font-style: normal;
+    font-weight: normal;
+    src: url('/media/fonts/FiraSans-Heavy.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-Heavy.woff') format('woff');
+}
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+$color-photon-blue: #0099db;
+
+#outer-wrapper {
+    background: #fff;
+}
+
+.content-section {
+    color: #000;
+    padding: 40px 0;
+    text-align: center;
+
+    @media #{$mq-desktop} {
+        padding: 50px 0;
+        text-align: left;
+    }
+}
+
+a.button:link.button-orange,
+a.button:visited.button-orange {
```

Comment:
I know this is just following the design, but I honestly think the page looks OK with our regular green download button (it may even stand out a little more). Just a pause for thought 🤔